README.md

```diff
@@ -43,3 +43,17 @@ npm run dev
 ```
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+
+### Fix to errors
+
+If you get following error `dotenv: not found` while trying to run command 
+```bash
+npm run prisma
+```
+make sure you have installed `dotenv` package globally. You can install the `dotenv` package using following command
+```bash
+npm install -g dotenv-cli
+```
+
+If you're using Linux, you most likely will need root access level to install the package globally
```

Comment:
Tein tämmöisen kohdan `README.md`-tiedostoon. Uutta konetta käyttöön ottaessa tuli näitä ongelmia vastaan, joten ajattelin kirjata ne itselleni ylös, jos tulevaisuudessa sattuisikin tulemaan sama ongelma :smile:

Reply:
Olisiko tuo `dotenv-cli`-paketti itseasiassa järkevä lisätä `package.json`-tiedostoon? Se estänee tuon virheen toteutumisen

Kurkkasin `dotenv-cli`:n npm.js-sivun ja siellä luki `Installation`-kohdassa: `npm install -g dotenv-cli`

[https://www.npmjs.com/package/dotenv-cli](https://www.npmjs.com/package/dotenv-cli)

Reply:
Joo, lisään `dotenv-cli`:n package.json :iin :+1: